docs/scripts_md/ImagingUpload.md

```diff
@@ -0,0 +1,163 @@
+# NAME
+
+NeuroDB::ImagingUpload -- Provides an interface to the uploaded imaging file
+
+# SYNOPSIS
+
+    use NeuroDB::ImagingUpload;
+
+    my $imaging_upload = &NeuroDB::ImagingUpload->new(
+                           \$dbh,
+                           $TmpDir_decompressed_folder,
+                           $upload_id,
+                           $patient_name,
+                           $profile,
+                           $verbose
+                         );
+
+    my $is_candinfovalid = $imaging_upload->IsCandidateInfoValid();
+
+    my $output = $imaging_upload->runDicomTar();
+    $imaging_upload->updateMRIUploadTable('Inserting', 0) if ( !$output );
+
+
+    my $output = $imaging_upload->runTarchiveLoader();
+    $imaging_upload->updateMRIUploadTable('Inserting', 0) if ( !$output);
+
+    my $isCleaned = $imaging_upload->CleanUpDataIncomingDir($uploaded_file);
+
+# DESCRIPTION
+
+This library regroups utilities for manipulation of the uploaded imaging file
+and updates of the `mri_upload` table according to the upload status.
+
+## Methods
+
+### new($dbhr, $uploaded\_temp\_folder, $upload\_id, ...) (constructor)
+
+Create a new instance of this class. This constructor needs the location of
+the uploaded file. Once the uploaded file has been validated, it will be
+moved to a final destination directory.
+
+INPUT:
+  - $dbhr                : database handler
+  - $uploaded\_temp\_folder: temporary directory of the upload
+  - $upload\_id           : uploadID from the mri\_upload table
+  - $pname               : patient name
+  - $profile             : name of the configuration file
+                          (typically `prod`)
+
+RETURNS: new instance of this class
+
+### IsCandidateInfoValid()
+
+Validates the File to be uploaded. If the validation passes, the following
+actions will happen:
+  1) Copy the file from tmp folder to /data/incoming
```

Comment:
maybe 
a) change the numbering to hyphens (to be consistent with other methods when we list actions)
b) instead of "Copy the file'; can we say 'The file will be copied from the `/tmp/` folder to ...'

Reply:
is it please possible to make the `/tmp` and `/data/incoming/` as code?